### PR TITLE
lab1b: add lexic, syntax and token in BNF

### PR DESCRIPTION
**Pull request by Daniel-Todasca**

### CODE REVIEW

File: lab1b/syntax.in [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<instr>::=<decl>|<logic_instr>|<comm>
+<program>::=<instr>|<instr><new_line><program>
+
+<comm>::=#<any_sequence>
+<any_sequence>::=<any_character>|<any_character><any_sequence>
+<any_character>::=<letter>|<digit>|<operators>|<separators>
+
+<decl>::=<var_decl>|<arr_decl>
+<var_decl>::=<TYPE><space><identifier>
+<TYPE>::=i|s
```
Comment (Tasadan-Filip) [inline lab1b/syntax.in]: You should have a type for a single character

File: lab1b/lexic.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+c. constants
+1. integer
+<i_constant>::=0|-<i_unsigned>|+<i_unsigned>|<i_unsigned>
+<i_unsigned>::=<nz-digit>|<nz_digit><dig_sequence>
+<dig_sequence>::=<digit>|<digit><dig_sequence>
+
+2. strings
+<s_constant>::=""|"<c_sequence>"
+<c_sequence>::=<char>|<char><c_sequence>
+<char>::=<letter>|<digit>|-
```
Comment (Tasadan-Filip) [inline lab1b/lexic.txt]: You can add also underscore as a separator

File: lab1b/syntax.in [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+<expression>::=<term>|(<expression>)|<expression><math_op><expression>
+<math_op>::=+|-|/|*|**|%
+<term>::=<identifier>|<i_constant>
+
+<io_stmt>::=in(<identifier>)|out(<identiier>)
+
+<if_stmt>::=if<condition>:<program>.|if<condition>:<program>.el:<program>.
+<condition>::=<expression><relation><expression>
+<relation>::=<|>|==|!=|>=|<=
```
Comment (Tasadan-Filip) [inline lab1b/syntax.in]: What is the difference between =< and <=?

**Review by Tasadan-Filip — APPROVED** [review]

File: lab1b/syntax.in (hunk 3 above)
Comment (Daniel-Todasca) [reply lab1b/syntax.in]: unclear formatting, will change

File: lab1b/lexic.txt (hunk 2 above)
Comment (Daniel-Todasca) [reply lab1b/lexic.txt]: will add in alphabet in next commit

**Pull request merged**